runtime/src/tasks/fifo/tasks-fifo.c

```diff
@@ -496,6 +497,17 @@ void chpl_task_addToTaskList(chpl_fn_int_t fid, void* arg,
 
   assert(subloc == 0 || subloc == c_sublocid_any);
 
+  // Visual Debug -- should be protected by an #ifdef VISUALDEBUG?
+  if (chpl_vdebug) {
```

Comment:
if 'chpl_vdebug' is a const int ==0 when VISUALDEBUG is not defined then the C compiler can evaulate the if-stmt and remove the body at compile time, which would obviate the need for an ifdef here (or anywhere else, I would think).